### DISCUSSION TITLE
Export MA playlist

### DISCUSSION THREAD

**Discussion by Simpelton** [music-assistant/support#2756] [Q&A] [answered]
Is it possible to export a playlist from MA to a file like .M3U?

My thinking is to export my playlists from youtube music so I can import them into a different music streaming provider. The streaming providers try to lock me in and the available tools to migrate playlists from one provider to an other are usually pretty shady or costly (for a one time usage).

Naturally, a playlist transfer from one streaming provider to an other would even be better ;-)

**Comment by marcelveldt** [comment, marked as answer]
we already support transferring playlists from one provider to another. just select all tracks of one playlist and then add them to another playlist

**Reply by fuslwusl** [reply]
Hi @marcelveldt, it would be great if you could simply export an entire playlist via the menu. The current option to export only the selected songs is very tedious with long playlists. 
It would also be useful to have a function in the UI to select all songs with one click (e.g. in the header as a checkbox).
Do you think this would be possible with little effort? You could then easily switch from one provider to another via Music Assistant. This is otherwise only possible via paid online services to which you then transfer your data, as far as I know.

I'm currently trying to move from Deezer to Tidal and currently have no option other than to select and export thousands of songs individually. 
With the ability to easily export playlists or even all albums or artists, you could use Music Assistant as a relocation helper.

Thanks for your great work!

**Reply by marcelveldt** [reply]
You can simply do CTRL A to select all items
